### PR TITLE
Fix zypper_in test for maintainanc

### PR DESCRIPTION
**Pull request by aylen384**
Fix tests like https://openqa.suse.de/tests/8253225#step/zypper_in/26

Verification: http://kazhua.qa.suse.de/tests/65

### CODE REVIEW

**Comment by DimStar77** [comment]
incomplete - alsa-devel is also a problem in multiple tests

File: tests/console/zypper_in.pm [hunk 1]
```diff
@@ -56,7 +56,7 @@ sub run {
     script_run 'zypper -n in --allow-unsigned-rpm ' . data_url('zypper/pa.rpm');
     script_run 'rm -f /preinstall_fail';
 
-    assert_script_run 'zypper -n in adaptec-firmware adobe-sourcecodepro-fonts alsa-devel apache2 atmel-firmware';
+    assert_script_run 'zypper -n in adaptec-firmware geoipupdate alsa-devel apache2 atmel-firmware';
```
Comment (DimStar77) [inline tests/console/zypper_in.pm:59]: https://openqa.opensuse.org/tests/2220481#step/zypper_in/26 - alsa-devel is amongst the not-always-existing packages

Comment (DimStar77) [reply tests/console/zypper_in.pm]: or SLE test: https://openqa.suse.de/tests/8252862#step/zypper_in/26

**Comment by dzedro** [comment]
No you have to test it it on all versions and all architectures, I'm reverting the original PR.

File: tests/console/zypper_in.pm [hunk 2]
```diff
@@ -56,7 +56,7 @@ sub run {
     script_run 'zypper -n in --allow-unsigned-rpm ' . data_url('zypper/pa.rpm');
     script_run 'rm -f /preinstall_fail';
 
-    assert_script_run 'zypper -n in adaptec-firmware adobe-sourcecodepro-fonts alsa-devel apache2 atmel-firmware';
+    assert_script_run 'zypper -n in adaptec-firmware geoipupdate apache2 atmel-firmware';
```
[the remark this answers is not in the document]
Comment (asdil12) [reply tests/console/zypper_in.pm]: Is there a way to find out what is on that staging DVDs?
I just want to install some random packages to see how zypper behaves when installing multiple packages.

**Pull request closed without merging**